src/main/resources/config/da.yml

```
@@ -0,0 +1,350 @@
+#
+# DiscordChatChannelConsoleCommandEnabled: om der skal tillades konsolkommandoer fra en chatkanal.
+# DiscordChatChannelConsoleCommandNotifyErrors: om man skal sende en bruger, der forsøger at køre en kommando uden tilladelse, at de ikke har tilladelse
+# DiscordChatChannelConsoleCommandPrefix: præfiks til brug for konsolkommandoer. f.eks. "!c tps"
+# DiscordChatChannelConsoleCommandRolesAllowed: de brugerroller, der har tilladelse til at udføre serverkommandoer fra chatkanalen
+# DiscordChatChannelConsoleCommandWhitelist: liste over kommandoer, der kan køres med DiscordChatChannelConsoleCommandPrefix
+# DiscordChatChannelConsoleCommandWhitelistBypassRoles: liste over roller, der omgår hvidlisten
+# DiscordChatChannelConsoleCommandWhitelistActsAsBlacklist: skal kommandoens hvidliste fungere som en sortliste i stedet
+# DiscordChatChannelConsoleCommandExpiration: tid i sekunder, indtil et sendt kommandooutput automatisk fjernes af botten. sat til 0 for at deaktivere udløb.
+# DiscordChatChannelConsoleCommandExpirationDeleteRequest: om meddelelsen fra den person, der oprindeligt udstedte kommandoen, skal slettes
+#
+DiscordChatChannelConsoleCommandEnabled: true
+DiscordChatChannelConsoleCommandNotifyErrors: true
+DiscordChatChannelConsoleCommandPrefix: "!c"
+DiscordChatChannelConsoleCommandRolesAllowed: ["Udvikler", "Ejer"]
+DiscordChatChannelConsoleCommandWhitelist: ["say", "lag", "tps"]
+DiscordChatChannelConsoleCommandWhitelistBypassRoles: ["Ejer", "Udvikler"]
+DiscordChatChannelConsoleCommandWhitelistActsAsBlacklist: false
+DiscordChatChannelConsoleCommandExpiration: 0
+DiscordChatChannelConsoleCommandExpirationDeleteRequest: true
+
+# Chat kanal af spillerliste kommando
+# Konfiguration til playerlist-kommandoen
+#
+# DiscordChatChannelListCommandEnabled: om kommandoen er aktiveret
+# DiscordChatChannelListCommandMessage: kommandoen folk kan skrive for at få spillerlisten
+# DiscordChatChannelListCommandExpiration: tid i sekunder, indtil en sendt afspillerlistebesked automatisk fjernes af botten. sat til 0 for at deaktivere udløb.
+# DiscordChatChannelListCommandExpirationDeleteRequest: om meddelelsen fra den person, der oprindeligt anmodede om spillerlisten skal slettes
+#
+DiscordChatChannelListCommandEnabled: true
+DiscordChatChannelListCommandMessage: "playerlist"
+DiscordChatChannelListCommandExpiration: 10
+DiscordChatChannelListCommandExpirationDeleteRequest: true
+
+# Chat kanal blacklisted sætninger og regulært udtryk
+#
+# DiscordChatChannelGameFilters: regex-filtre, der skal anvendes på chatbeskeder, der sendes til Discord, hvis resultatet er tomt, vil beskeden slet ikke blive sendt
+# DiscordChatChannelDiscordFilters: regex-filtre, der skal anvendes på chatbeskeder, der sendes til Minecraft, hvis resultatet er tomt, vil beskeden slet ikke blive sendt
+#
+DiscordChatChannelGameFilters: {}
+DiscordChatChannelDiscordFilters: {".*Online spillere \\(.*": "", ".*\\*\\*Ingen online spillere\\*\\*.*": ""}

```

```suggestion
DiscordChatChannelDiscordFilters: {".*Online players \\(.*": "", ".*\\*\\*No online players\\*\\*.*": ""}

```
The output will always be English since it's hardcoded in the bukkit server software.